src/games/strategy/triplea/ai/proAI/data/ProMyMoveOptions.java

```diff
@@ -56,4 +59,8 @@ public ProMyMoveOptions(final ProMyMoveOptions myMoveOptions) {
     return transportList;
   }
 
+  public Map<Unit, Set<Territory>> getBomberMoveMap() {
```

Comment:
I'm not necessarily recommending to move any code other than what is listed, just about 30 lines. Part of it is decomposing the state into some sort of abstraction, so it does not all have to be so low level and the state represented mostly as primitives.

Ideally things would be broken up into a number of modules. So it will be weird to have just a piece of this large class broken up when most of it is not. But that is on the right path, and makes the decomposed stuff simpler, which is easier to understand, less buggy (and testable)